src/__mocks__/transaction-calldata.mock.ts

```diff
@@ -0,0 +1,168 @@
+import { faker } from '@faker-js/faker';
+import { ethers } from 'ethers';
+import {
+  getMultiSendCallOnlyDeployment,
+  getProxyFactoryDeployment,
+  getSafeSingletonDeployment,
+} from '@safe-global/safe-deployments';
+import type { DeploymentFilter } from '@safe-global/safe-deployments';
+
+export const MOCK_UNKNOWN_CALL_DATA = ethers.zeroPadBytes('0x7a761202', 970);
```

Comment:
Nit: I think `MOCK_UNSUPPORTED_CALLDATA` would match a bit better.

What exactly is this data and why is it padded by 970?

Reply:
I've renamed this and created a fake function:

```ts
const MOCK_UNSUPPORTED_CALLDATA = new ethers.Interface([
  'function unsupported()',
]).encodeFunctionData('unsupported');
```